ci/scripts/jenkins/checks.sh

```diff
@@ -37,7 +37,7 @@ show_conda_info
 gpuci_logger "Installing IWYU"
 git clone https://github.com/include-what-you-use/include-what-you-use.git ${IWYU_DIR}
 pushd ${IWYU_DIR}
-git checkout clang_12
+git checkout clang_14
```

Comment:
With clang-14, we can now use a conda `include-what-you-use` package. Will create a new issue to update to using that package instead of build/install manually.

Reply:
Do you want to just open a PR and target this PR?

Reply:
I've never tested the condo IWYU package, but this one I know works.

I think we should incrementally update IWYU to the condo package as part of a follow on PR.

Reply:
This will get fixed with issue #163